src/antares/model/binding_constraint.py

```diff
@@ -120,6 +195,14 @@ def id(self) -> str:
     def properties(self) -> BindingConstraintProperties:
         return self._properties
 
+    @properties.setter
```

Comment:
Not really familiar with this but isn't it a bit weird to have both a property and a property setter ? Shouldn't be just remove the private `_` then ?

Reply:
Probably, I believe I might have forgotten the case where we need to update the local properties from the new properties here and that would be the reason for the setter

Reply:
Corrected